<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 #bcbcbc, stop:1 #848484);

}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true">QStackedWidget#stackedwidget{
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 #bcbcbc, stop:1 #848484);

}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="7" column="1">
             <widget class="QPushButton" name="pushButton_admin_login">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>24</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 #00d89d, stop:1 #007868);

color:white;</string>
              </property>
              <property name="text">
               <string>Sign in</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 rgba(0, 184, 157, 255), stop:1 rgba(0, 120, 103, 255));

color:white;</string>
              </property>
              <property name="text">
               <string>Login</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>36</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 rgba(0, 184, 157, 255), stop:1 rgba(0, 120, 103, 255));
border-radius: 15px;
color:white;</string>
              </property>
              <property name="text">
               <string>Admin panel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_admin_login">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 rgba(0, 184, 157, 255), stop:1 rgba(0, 120, 103, 255));

color:white;</string>
              </property>
              <property name="text">
               <string>Password</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="lineEdit_admin_password">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="0">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="4" column="1">
             <widget class="QPushButton" name="pushButton_withdraw">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>24</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 #00d89d, stop:1 #007868);
border-radius: 15px;
color:white;</string>
              </property>
              <property name="text">
               <string>Withdraw cash</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_balance">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>24</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 #00d89d, stop:1 #007868);
border-radius: 15px;
color:white;</string>
              </property>
              <property name="text">
               <string>Show ATM balance</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_refill">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>24</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 #00d89d, stop:1 #007868);
border-radius: 15px;
color:white;</string>
              </property>
              <property name="text">
               <string>Refill ATM</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
